include/StochasticNoiseSimulator.hpp

```diff
@@ -127,6 +127,11 @@ class StochasticNoiseSimulator: public Simulator {
     [[nodiscard]] dd::NoiseOperationKind ReturnNoiseOperation(char i, double d) const;
 
     [[nodiscard]] std::string intToString(long target_number) const;
+
+    double ApproximateEdgeByFidelity(std::unique_ptr<dd::Package>& localDD, dd::Package::vEdge& edge, double targetFidelity, bool allLevels, bool removeNodes);
+
+    dd::Package::vEdge RemoveNodesInPackage(std::unique_ptr<dd::Package>& localDD, dd::Package::vEdge e, std::map<dd::Package::vNode*, dd::Package::vEdge>& dag_edges);
```

Comment:
This is available in the dd package now.

Reply:
Removed unused functions